helm/trivy/values.yaml

```diff
@@ -7,6 +7,9 @@ image:
   tag: 0.14.0
   pullPolicy: IfNotPresent
 
+imagePullSecrets: []
```

Comment:
IIUC This secret is used to pull the Trivy container image, which defaults to docker.io/aquasec/trivy:0.14.0.

Is there any reason to have an array here?

Maybe we could also explain how this secret is used. For someone who does not know the implementation details they may think that we use it to scan private container images.

Reply:
From https://kubernetes.io/docs/concepts/containers/images/,
```
cat <<EOF > pod.yaml
apiVersion: v1
kind: Pod
metadata:
  name: foo
  namespace: awesomeapps
spec:
  containers:
    - name: foo
      image: janedoe/awesomeapp:v1
  imagePullSecrets:
    - name: myregistrykey
EOF

cat <<EOF >> ./kustomization.yaml
resources:
- pod.yaml
EOF
```

`imagePullSecrets` is designed array originally.
I think user can write two or more secrets for docker hub and private repos.
```
imagePullSecrets:
  - name: secret1
  - name: secret2
```

Reply:
Yup. I know that in Kubernetes you can specify an array of local references to secrets. However, in case of this Helm chart there's only **one** image docker.io/aquasec/trivy:0.14.0. What's more exposing an option to specify > 1 secrets might be confusing. Which secret is used in you example when both refer to docker.io registry server?

Reply:
Aha, now only matter is docker hub.

Reply:
I changed documentation and the variable to make it more clear, that the secret is only used for the trivy image.